src/Transports.AspNetCore/GraphQLHttpMiddleware.cs

```diff
@@ -298,7 +298,11 @@ protected virtual ValueTask<bool> HandleAuthorizeWebSocketConnectionAsync(HttpCo
             if (result.Errors?.Any(e => e is HttpMethodValidationError) == true)
                 statusCode = HttpStatusCode.MethodNotAllowed;
             else if (_options.ValidationErrorsReturnBadRequest)
+            {
                 statusCode = HttpStatusCode.BadRequest;
+                if (!string.IsNullOrEmpty(_options.ValidationErrorsReturnHeader))
+                    context.Response.Headers.Add(_options.ValidationErrorsReturnHeader, "true");
```

Comment:
I doubt about `true` value but without any value I don't see added header into response.

Reply:
Another option - add delegate to options class instead of string but it looks complicated.